[bug] Except in proc_msg

[ERROR] Except in proc_msg
```
Traceback (most recent call last):
  File "/home/liuyang/wall/weixin/wxbot2/lib/wxbot.py", line 613, in proc_msg
    self.handle_msg(r)
  File "/home/liuyang/wall/weixin/wxbot2/lib/wxbot.py", line 583, in handle_msg
    content = self.extract_msg_content(msg_type_id, msg)
  File "/home/liuyang/wall/weixin/wxbot2/lib/wxbot.py", line 416, in extract_msg_content
    name = self.get_group_member_prefer_name(self.get_group_member_name(uid, msg['FromUserName']))
TypeError: get_group_member_name() takes exactly 2 arguments (3 given)
```

--- a/wxbot.py
+++ b/wxbot.py
@@ -3,6 +3,7 @@
 
 import os
 import sys
+import traceback
 import webbrowser
 import pyqrcode
 import requests
@@ -39,13 +40,17 @@ def show_image(file_path):
         webbrowser.open(file_path)
 
 class SafeSession(requests.Session):
-    def request(self, *args, **kwargs):
+    def request(self, method, url, params=None, data=None, headers=None, cookies=None, files=None, auth=None,
+                timeout=None, allow_redirects=True, proxies=None, hooks=None, stream=None, verify=None, cert=None,
+                json=None):
         for i in range(3):
             try:
-                return super(SafeSession, self).request(*args, **kwargs)
-            except:
-                pass
-        return super(SafeSession, self).request(*args, **kwargs)
+                return super(SafeSession, self).request(method, url, params, data, headers, cookies, files, auth,
+                                                        timeout,
+                                                        allow_redirects, proxies, hooks, stream, verify, cert, json)
+            except Exception as e:
+                print e.message,traceback.format_exc()
+                continue
 
 
 class WXBot:
@@ -241,6 +246,7 @@ class WXBot:
             return None
         else:
             return name
+
 
 
 
@@ -475,7 +481,9 @@ class WXBot:
                                    'title': msg['FileName'],
                                    'desc': self.search_content('des', content, 'xml'),
                                    'url': msg['Url'],
-                                   'from': self.search_content('appname', content, 'xml')}
+                                   'from': self.search_content('appname', content, 'xml'),
+                                   'content':msg.get('content')#有的公众号会发一次性3 4条链接一个大图,如果只url那只能获取第一条,content里面有所有的链接
+                                   }
             if self.DEBUG:
                 print '    %s[Share] %s' % (msg_prefix, app_msg_type)
                 print '    --------------------------'
@@ -483,6 +491,7 @@ class WXBot:
                 print '    | desc: %s' % self.search_content('des', content, 'xml')
                 print '    | link: %s' % msg['Url']
                 print '    | from: %s' % self.search_content('appname', content, 'xml')
+                print '    | content: %s' % msg.get('content')[:20]
                 print '    --------------------------'
 
         elif mtype == 62:

--- a/wxbot.py
+++ b/wxbot.py
@@ -140,7 +140,6 @@ class WXBot:
                 self.account_info['normal_member'][contact['UserName']] = {'type': 'group', 'info': contact}
             elif contact['UserName'] == self.my_account['UserName']:  # 自己
                 self.account_info['normal_member'][contact['UserName']] = {'type': 'self', 'info': contact}
-                pass
             else:
                 self.contact_list.append(contact)
                 self.account_info['normal_member'][contact['UserName']] = {'type': 'contact', 'info': contact}
@@ -243,22 +242,6 @@ class WXBot:
         else:
             return name
 
-    def get_group_member_name(self, uid):
-        info = self.get_group_member_info(uid)
-        if info is None:
-            return None
-        info = info['info']
-        name = {}
-        if 'RemarkName' in info and info['RemarkName']:
-            name['remark_name'] = info['RemarkName']
-        if 'NickName' in info and info['NickName']:
-            name['nickname'] = info['NickName']
-        if 'DisplayName' in info and info['DisplayName']:
-            name['display_name'] = info['DisplayName']
-        if len(name) == 0:
-            return None
-        else:
-            return name
 
 
     @staticmethod

--- a/wxbot.py
+++ b/wxbot.py
@@ -482,7 +482,7 @@ class WXBot:
                                    'desc': self.search_content('des', content, 'xml'),
                                    'url': msg['Url'],
                                    'from': self.search_content('appname', content, 'xml'),
-                                   'content':msg.get('content')#有的公众号会发一次性3 4条链接一个大图,如果只url那只能获取第一条,content里面有所有的链接
+                                   'content':msg.get('Content')#有的公众号会发一次性3 4条链接一个大图,如果只url那只能获取第一条,content里面有所有的链接
                                    }
             if self.DEBUG:
                 print '    %s[Share] %s' % (msg_prefix, app_msg_type)

